### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by sharonwarner**
login

Screen Shot 2020-06-25 at 3 12 54 PM

===

Resolves #XXX

- [x] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "edd-calculator": "https://www.edd.ca.gov/Unemployment/ui-calculator.htm",
     "edd-cares": "https://edd.ca.gov/about_edd/coronavirus-2019/cares-act.htm",
     "edd-pua": "https://edd.ca.gov/about_edd/coronavirus-2019/pandemic-unemployment-assistance.htm",
-    "edd-webapp": "https://portal.edd.ca.gov/WebApp/Login?resource_url=https://portal.edd.ca.gov/WebApp/Home",
+    "edd-webapp": "https://edd.ca.gov/Unemployment/UI_Online.htm",
```
Comment (idealisms) [inline public/locales/en/translation.json:55]: This link is shared with the guide! I think we need a new entry for the retro-certs link.

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -333,8 +333,8 @@
     "ack-list-item-2": "I know the law provides penalties if I make false statements or withhold facts to receive benefits; and my answers are true and correct.",
     "ack-list-item-3": "I declare under penalty of perjury that I am a US Citizen or National; or an Alien in satisfactory immigration status and permitted to work by the United States Citizenship and Immigration Service.",
     "ack-list-item-4": "I understand when submitting my request for benefits my submission is considered the same as my written signature.",
-    "ack-label": "I agree and accept the statement above.",
-    "ack-feedback": "You must indicate your acceptance of the statement by checking the box before your certification can be submitted.",
+    "ack-label": "You must indicate your acceptance of the statement by checking the box before your certification can be submitted.",
+    "ack-feedback": "This field is required.",
```
Comment (idealisms) [inline public/locales/en/translation.json:337]: You can remove this and use "required-error".

File: public/locales/es/translation.json [hunk 3]
```diff
@@ -52,7 +52,7 @@
     "edd-calculator": "https://www.edd.ca.gov/Unemployment/ui-calculator.htm",
     "edd-cares": "https://edd.ca.gov/about_edd/coronavirus-2019/cares-act.htm",
     "edd-pua": "https://edd.ca.gov/about_edd/coronavirus-2019/pandemic-unemployment-assistance.htm",
-    "edd-webapp": "https://portal.edd.ca.gov/WebApp/Login?resource_url=https://portal.edd.ca.gov/WebApp/Home",
+    "edd-webapp": "https://edd.ca.gov/Unemployment/UI_Online.htm",
```
Comment (idealisms) [inline public/locales/es/translation.json:55]: https://edd.ca.gov/Unemployment/UI_Online_Espanol.htm

File: public/locales/en/translation.json (hunk 1 above)
Comment (sharonwarner) [reply public/locales/en/translation.json:55]: oof good call

**Pull request merged**